process/sync/metablock.go

```diff
@@ -204,6 +214,14 @@ func (boot *MetaBootstrap) requestHeaderWithHash(hash []byte) {
 	err := boot.hdrRes.RequestDataFromHash(hash)
 	if err != nil {
 		log.Debug("RequestDataFromHash", "error", err.Error())
+		return
+	}
+
+	boot.requestedItemsHandler.Sweep()
```

Comment:
these sweeps should not be needed.

Reply:
For now, until the refactor of RequestHandler which would replace ResolverFinder in sync sweeps should be done from here (anyway this method and others like this where sweep is called in sync, are called only one time per round).